### PR TITLE
 SETUP: Initial setup

### PR DESCRIPTION
**Pull request by sahsisunny**
**Description**
This PR creates a new Next.js project and sets up the development environment. 

**The following tools are configured:**
- [x] ESLint for linting
- [x] Tailwind CSS for styling
- [x] Prettier for code formatting
- [x] Husky for pre-commit checks
- [x] Volta for version management
- [x] TypeScript for type safety

These tools provide a solid foundation for the project's front-end development.

### Video Recording

https://github.com/Real-Dev-Squad/tiny-site-frontend/assets/82165483/e95905ae-9e9b-4a17-a68a-64774b9852da

### CODE REVIEW

**Review by iamitprakash — CHANGES_REQUESTED** [review]
test case stats are missing

**Comment by sahsisunny** [comment]
> test case stats are missing 

I have already created an issue ticket for adding a test framework since it's currently not included.

**Comment by iamitprakash** [comment]
@sahsisunny please add working application video or PIC

**Review by Pratiyushkumar — COMMENTED** [review]
agreed with @iamitprakash, you should add video of working application

File: public/vercel.svg
Comment (Pratiyushkumar) [inline public/vercel.svg:1]: remove this svg if not needed

File: public/next.svg
Comment (Pratiyushkumar) [inline public/next.svg:1]: remove this svg if not required

File: src/app/favicon.ico
Comment (Pratiyushkumar) [inline src/app/favicon.ico:1]: also this favicon

File: src/app/page.tsx
Comment (Pratiyushkumar) [inline src/app/page.tsx:1]: we could have removed these unwanted code.

**Review by Pratiyushkumar — COMMENTED** [review]
Request you remove unwanted  code from .css files and.tsx files

**Comment by sahsisunny** [comment]
@Pratiyushkumar default templates will be removed when we make other components and pages.

**Review by vinit717 — APPROVED** [review]

**Review by Pratiyushkumar — APPROVED** [review]

**Pull request merged**